packages/containers/src/List/selector.js

```diff
@@ -1,7 +1,7 @@
 import { createSelector } from 'reselect';
 import { Map, List } from 'immutable';
 
-export function contains(ob, query) {
+function contains(ob, query) {
```

Comment:
I haven't touched that code except the export. Looking at the code it seems that this is a list item, so any object. I'll rename it to `listItem`